stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -739,85 +728,205 @@ end
 
 #Linear solvers
 #Generic solver using naive substitution
-function ldiv!(A::Bidiagonal, b::AbstractVector)
-    require_one_based_indexing(A, b)
+ldiv!(A::Bidiagonal, b::AbstractVecOrMat) = @inline ldiv!(b, A, b)
+function ldiv!(c::AbstractVecOrMat, A::Bidiagonal, b::AbstractVector)
```

Comment:
Couldn't `b` be `AbstractVecOrMat` here, and support `size(c,2) == size(b,2) > 1`?